### PR TITLE
Add endpoints to get Census Response Rates

### PR DESCRIPTION
**Pull request by alanren2**
PR that allows for a GET endpoint either providing year or date and getting all the tracts' response rates.

Example Response:
```
{
    "message": "",
    "result": {
        "response_rates": [
            {
                "rate": 0,
                "tract_id": "17031311100"
            },
            {
                "rate": 0.13,
                "tract_id": "20209042502"
            }, ...
        ]
    },
    "success": true
}
```

### CODE REVIEW

**Comment by coconut750750** [comment]
what are these commits?!?!?!?!?!

File: backend/api/views/main.py [hunk 1]
```diff
@@ -24,6 +24,30 @@ def get_persons():
     return create_response(data={"persons": persons})
 
 
+# function that is called when you visit /census_response
+@main.route("/census_response", methods=["GET"])
+def get_census_response():
```
Comment (caiyingshen) [inline backend/api/views/main.py:29]: could you make the endpoint name more descriptive of what the query & params are?

**Review by chyku — COMMENTED** [review]
btw can you put an example response in the description?

**Review by caiyingshen — APPROVED** [review]
😱

File: backend/api/views/main.py [hunk 2]
```diff
@@ -24,6 +24,30 @@ def get_persons():
     return create_response(data={"persons": persons})
 
 
+# function that is called when you visit /census_response
+@main.route("/census_response_by_datetime", methods=["GET"])
```
Comment (chyku) [inline backend/api/views/main.py]: can we make the url something like `/census_response?datetime="jdskflsdjkl"` bc this endpoint name is tilting me

**Review by chyku — CHANGES_REQUESTED** [review]

**Review by chyku — APPROVED** [review]

**Pull request merged**